deep_qa/models/multiple_choice_qa/tuple_inference.py

```diff
@@ -175,3 +177,99 @@ def _build_model(self):
         final_output = MaskedSoftmax()(options_probabilities)
 
         return DeepQaModel(input=[question_input, background_input], output=[final_output])
+
+    def debug_display_instance_tuple_match_scores(self, answer_tuples, background_tuples, tuple_matcher_output,
```

Comment:
Minor point, feel free to ignore.  I would probably put this method below the main `_instance_debug_output`.  Also, the method name should probably begin with an underscore, and giving types on the input arguments would help readability here.